wiki/Modding/Nuke/id.md

```diff
@@ -0,0 +1,21 @@
+---
+stub: true
+---
+
+# Nuke
+
+**Nuke** adalah sebuah alat moderasi [beatmap](/wiki/Beatmaps) di tahun-tahun awal osu!. Beatmap apa pun yang dianggap mengabaikan aturan (atau kreator mengabaikan [Kode Etik](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)) bisa di nuke oleh [Beatmap Appreciation Team (BAT)](/wiki/Modding/Beatmap_Appreciation_Team) atau [Global Moderation Team (GMT)](/wiki/People/The_Team/Global_Moderation_Team).
```

Comment:
```suggestion
**Nuke** adalah sebuah alat moderasi [beatmap](/wiki/Beatmaps) di tahun-tahun awal osu!. Beatmap apa pun yang dianggap mengabaikan aturan (atau kreator mengabaikan [Kode Etik](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)) bisa di-nuke oleh [Beatmap Appreciation Team (BAT)](/wiki/Modding/Beatmap_Appreciation_Team) atau [Global Moderation Team (GMT)](/wiki/People/The_Team/Global_Moderation_Team).
```
verb shouldnt be separated

Reply:
ahh ya lupa pake strip. didn't notice this :D nanti semuanya di recheck lagi